spring-boot-starter-camunda/src/main/resources/application-camunda-simple.yaml

```diff
@@ -9,6 +9,7 @@ camunda:
     zeebe:
       enabled: true
       base-url: http://localhost:26500
+      rest-address: http://localhost:8080
```

Comment:
This needs to be aligned with the docker-compose from camunda/camunda-platform

Reply:
indeed, it is still not defined which port will be used for Zeebe REST in camunda-platform